apps/full-site-editing/full-site-editing-plugin/full-site-editing/class-full-site-editing.php

```diff
@@ -16,6 +16,8 @@ class Full_Site_Editing {
 	 */
 	private static $instance = null;
 
+	private $template_post_types = [ 'wp_template', 'wp_template_part', 'wp_template_part_type' ];
```

Comment:
So far we've been sticking to `array()` format due to compat concerns. Should probably stick to that unless those compat concerns are no longer relevant (which is likely given our rescoping), in which case we should open a separate PR that just converts them all over.

Reply:
Fixed on 67c9e0879c941743f2a96649b89a16d4f8f941cd.